src/routes/backpack/logmanager.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+|--------------------------------------------------------------------------
+| Backpack\LogManager Routes
+|--------------------------------------------------------------------------
+|
+| This file is where you may define or change all of the routes that are
+| handled by the Backpack\LogManager package.
+|
+*/
+
+Route::group([
+            'namespace'  => 'Backpack\LogManager\app\Http\Controllers',
```

Comment:
I think so, yes. Would change this, but I'm sure StyleCI would change it back to how it likes it :-)